internal/provider/schemas_test.go

```diff
@@ -282,3 +282,72 @@ func TestGeneratorDataSourceSchemas_ModelsBytes(t *testing.T) {
 		})
 	}
 }
+
+func TestGeneratorSchemas_SchemaBytes(t *testing.T) {
```

Comment:
Thanks for covering this with a test! We've recently added some more "end-to-end" golden file tests that I think may be a better fit for testing your changes.

Would you mind adding a `description` and `markdown_description` into the IR JSON file at `internal/cmd/testdata/custom_and_external/ir.json`, then run `make testdata`? It should update a bunch of golden files with the description/markdown description by running the whole generator.

After you do that, I think we can probably remove this test and the associated `.txt` files.